wcivf/apps/elections/models.py

```diff
@@ -49,9 +49,37 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @property
     def in_past(self):
+        """
+        Returns a boolean for whether the election date is in the past
+        """
         return self.election_date < datetime.date.today()
 
+    @property
+    def is_city_of_london(self):
+        """
+        Returns boolean for if the election is within City of London district
+        """
+        return "local.city-of-london" in self.slug
+
+    @property
+    def polls_close(self):
+        """
+        Return the time the polls close
+        """
+        if self.is_city_of_london:
+            return datetime.time(20, 0)
```

Comment:
Would it be possible to add a comment, maybe with a link to a page on the City of London website citing this? @pmk01 do you know of a good page?

Reply:
This? https://www.cityoflondon.gov.uk/about-us/voting-elections/elections/ward-elections

Reply:
The only place I could find where the actual opening/closing times were explicitly mentioned as being different was here https://www.citymatters.london/city-london-voters-may-go-polls-twice-2021/ there's a paragraph
 which states:

"This is because City elections open at 8am and close at 8pm, whilst GLA polling runs from 7am  at 10pm, so it would be impractical to share the same venue. An Act of Parliament dating back to 1957 would have to be changed to change the hours in the Square Mile."

Reply:
I'll link to both Peter's link and this old blog post of ours that references it https://democracyclub.org.uk/blog/2017/03/22/eight-weird-things-about-tomorrows-city-london-elections/